tests/redirects/map_globalconf.py

```diff
@@ -1094,4 +1094,9 @@
 
     # Bug 1313023
     url_test('/story', 'https://donate.mozilla.org/?source=story_redirect'),
+
+    # Bug 1277196
+    url_test('/firefox/{49.0,49.0.1,50.0a1,51.0a2}/firstrun/learnmore', 'firefox/features/'),
```

Comment:
Please also make sure we have a test for the URL without a version number (just `/firefox/firstrun/learnmore`). You seem to have deleted the original redirect for this.